core/src/components/datetime/datetime.scss

```diff
@@ -185,13 +185,36 @@ ion-picker-column-internal {
   display: none;
 }
 
-:host(.datetime-readonly),
 :host(.datetime-disabled) {
   pointer-events: none;
+
+  .calendar-days-of-week,
+  .datetime-time {
+    opacity: 0.4;
+  }
 }
 
-:host(.datetime-disabled) {
-  opacity: 0.4;
+:host(.datetime-readonly) {
+  pointer-events: none;
+
+  /**
+   * Allow user to navigate months
+   * while in readonly mode
+   */
+  .calendar-action-buttons,
+  .calendar-body,
+  .datetime-year {
+    pointer-events: initial;
+  }
+
+  /**
+   * Disabled buttons should have full opacity
+   * in readonly mode
+   */
+  .calendar-day[disabled],
+  .datetime-action-buttons ion-button[disabled] {
+    opacity: 1;
```

Comment:
Heads up that this does change the behavior of datetime slightly. If you have a date disabled using `isDateEnabled` it no longer appears dimmed.

Here is the behavior in `main`: https://codepen.io/liamdebeasi/pen/WNPbLXd (Feb 5 is dimmed since it is explicitly disabled)

If all the buttons are disabled do we still want to show the dimmed state for `isDateEnabled`? For screen readers it won't make much of a difference because those tools will announce every button as dimmed.

Reply:
I think we should still show it if possible, I can definitely see someone reporting this as a bug. We might need a class for when a date is disabled due to `isDateEnabled`, or something along those lines 🤔

Reply:
cool, i can make that change! (and add a screenshot test). i'm not really sure what sort of situations people are using readonly mode for, but it does make more sense to me that specifically-disabled dates would appear different, even when readonly.